src/FffPolygonGenerator.cpp

```diff
@@ -309,7 +309,7 @@ void FffPolygonGenerator::slices2polygons(SliceDataStorage& storage, TimeKeeper&
         std::multimap<int, size_t> order_to_mesh_indices;
         for (size_t mesh_idx = 0; mesh_idx < storage.meshes.size(); mesh_idx++)
         {
-            order_to_mesh_indices.emplace(storage.meshes[mesh_idx].settings.get<size_t>("infill_mesh_order"), mesh_idx);
+            order_to_mesh_indices.emplace(storage.meshes[mesh_idx].settings.get<int>("infill_mesh_order"), mesh_idx);
```

Comment:
It's not a big enough issue to stop me from merging this if I'm through with the component test, but I'm just curious to why you use `int` here. Or more specifically, why you'd replace `size_t`, and necessitating a new get method in the settings.

Reply:
A user might want to put a negative value. Would be combersome if you would have to increase all settings values of all other meshes if theres only one mesh you would like to have processed earlier than all the other.

Reply:
Ok, makes sense!